### PR TITLE
[stable/fluent-bit] Add priorityClassName

### PR DESCRIPTION
**Pull request by mmorejon**
#### What this PR does / why we need it:
It allows to set a priorityClassName.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 1.9.2
+version: 1.9.3
```
Comment (kfox1111) [inline stable/fluent-bit/Chart.yaml]: Its a new feature, so should be 1.10.0. Otherwise, looks good to me.

Comment (mmorejon) [reply stable/fluent-bit/Chart.yaml]: I'm agree with you.

**Review by kfox1111 — COMMENTED** [review]
/lgtm

**Pull request merged**